README.md

```diff
@@ -34,7 +34,7 @@ simdjson is easily consumable with a single .h and .cpp file.
      std::cout << tweets["search_metadata"]["count"] << " results." << std::endl;
    }
    ```
-3. `g++ -o parser parser.cpp` (or clang++)
+3. `g++ -o parser parser.cpp simdjson.cpp -std=c++17`
```

Comment:
could be `c++ -o parser parser.cpp simdjson.cpp -std=c++17` or `cpp -o parser parser.cpp simdjson.cpp -std=c++17` if you want to be more neutral with respect to which compiler is being used.

Reply:
Oh, I didn't realize there was a standard alias, and that clang supported -std=c++17 as an argument. Fixing.